packages/raga-app/tsconfig.json

```diff
@@ -4,6 +4,8 @@
   "compilerOptions": {
     "jsx": "react-jsx",
     "lib": ["dom", "es5", "es2015", "es2016", "es2017", "es2018", "es2019", "es2020"],
+    // HACKHACK due to https://github.com/table-library/react-table-library/issues/145
```

Comment:
filed this issue on react-table-library: https://github.com/table-library/react-table-library/issues/145